lmdeploy/pytorch/check_env/__init__.py

```diff
@@ -87,6 +87,27 @@ def check_env(device_type: str):
 MAX_TRANSFORMERS_VERSION = '4.41.2'
 
 
+def check_awq(hf_config):
+    """check awq support."""
+    logger = get_logger('lmdeploy')
+    quantization_config = getattr(hf_config, 'quantization_config', dict())
+    quant_method = quantization_config.get('quant_method', None)
+    if quant_method != 'awq':
+        return
+    try:
+        import awq  # noqa
+    except Exception as e:
+        _handle_exception(e, 'awq', logger)
+
+    try:
+        import awq_ext  # noqa
+    except Exception:
+        logger.debug('Exception:', exc_info=1)
+        logger.warning('Failed to import `awq_ext`. '
+                       'Try reinstall it from source: '
+                       'https://github.com/casper-hansen/AutoAWQ')
```

Comment:
install autoawq from source failed. 
```
  ERROR: HTTP error 404 while getting https://github.com/casper-hansen/AutoAWQ_kernels/releases/download/v0.0.7/autoawq_kernels-0.0.7+cu118-cp38-cp38-linux_x86_64.whl
ERROR: Could not install requirement autoawq-kernels@ https://github.com/casper-hansen/AutoAWQ_kernels/releases/download/v0.0.7/autoawq_kernels-0.0.7+cu118-cp38-cp38-linux_x86_64.whl from https://github.com/casper-hansen/AutoAWQ_kernels/releases/download/v0.0.7/autoawq_kernels-0.0.7+cu118-cp38-cp38-linux_x86_64.whl (from autoawq==0.2.5+cu118) because of HTTP error 404 Client Error: Not Found for url: https://github.com/casper-hansen/AutoAWQ_kernels/releases/download/v0.0.7/autoawq_kernels-0.0.7+cu118-cp38-cp38-linux_x86_64.whl for URL https://github.com/casper-hansen/AutoAWQ_kernels/releases/download/v0.0.7/autoawq_kernels-0.0.7+cu118-cp38-cp38-linux_x86_64.whl
```